handler.py

```diff
@@ -2,46 +2,45 @@
 from calculations.calculations import Calculations
 from plots.make_plot import make_plot
 import traceback
+import pandas as pd
 
-conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv")
+conversions = Conversions("data\\output2_linpot_2023-10-14_13-23-28.csv", "data\\output2_analog_2023-10-14_13-18-15.csv")
 calculations = Calculations("data\\output2_linpot_2023-10-14_13-23-28.csv")
+#conversions.clean_data(conversions.data)
 
 
 class handler:
 
     def wheel_load_handler(self):
         # loads and prepares initial data
-        conversions.convert_voltage_to_in()
-        conversions.convert_time(conversions.data)
-        conversions.clean_data()
-        calculations.data = conversions.data
+        # todo: return dataframe
+        df = conversions.data
+        df_xl = conversions.data2
+        df_converted = conversions.convert_voltage_to_in(df)
+        conversions.convert_xl_g()
+        df_cleaned = conversions.clean_data(df_converted)
+        calculations.data = df_cleaned
+        calculations.data2 = conversions.data2
+        df_xl_goal = calculations.data2
 
         # calculating constants
-        calculations.calculate_wheel_rate_front()
-        calculations.calculate_wheel_rate_rear()
-        calculations.calculate_wheel_rate_roll_front()
-        calculations.calculate_wheel_rate_roll_rear()
-        calculations.calculate_K_H_K_P()
-        calculations.calculate_K_R_K_W()
-        calculations.calculate_a_b()
-        calculations.calculate_q_w()
-
+        calculations.calculate_wheel_load_constants()
+        
         # calculates wheel load formulas in pounds of force
-        calculations.calculate_force_front_left()
-        calculations.calculate_force_front_right()
-        calculations.calculate_force_rear_left()
-        calculations.calculate_force_rear_right()
+        wheel_loads = calculations.calculate_wheel_loads(df_converted)
 
-        plots = make_plot(calculations.data)
-        # plots.plot_wheel_load_vs_acceleration()
+        conversions.convert_time(wheel_loads)
+        conversions.convert_time(df_xl_goal)
+        plots = make_plot(wheel_loads)
+        plots3 = make_plot(df_xl_goal)
         plots.plot_wheel_load_vs_time()
+        return plots3.plot_wheel_load_vs_acceleration(wheel_loads)
```

Comment:
return fig from make plot and use fig.show to have it be consistent with the rest of the codebase please